doc/iterations/2025.1/add_support_of_non_synchronized_tables.adoc

```diff
@@ -0,0 +1,40 @@
+= Add support of non synchronized tables
+
+== Problem
+
+Currently, tables are synchronized regarding the semantic elements found in the model.
+The table defines, inside its row description, what kind of elements the table handles and how they are retrieved.
+So currently, whenever a compatible element is created, it is automatically added as a new row of the table.
+There is no way to create a table that does not follow this mechanism. +
+It is not possible to let the user choose the elements displayed as rows in the table.
+
+== Key Result
+
+As a specifier, I want to describe a table that is not synchronized. +
+As a specifier, I want to define the drag & drop behavior that allows to add rows. +
```

Comment:
Technically, any operation should be able to add more rows. A drag and drop tool is convenient for this kind of interaction but an unsynchronized table could work without any drag and drop tools. Those are two separate subjects (synchronized and drag and drop tool)).